### ISSUE TITLE
[Hugo Migration] Add YouTube, Slides, Kaggle, Colab, and GitHub buttons

### ISSUE THREAD

**Issue by ionlights** [ucfai/ucfai#9]
Order in the title is arbitrary.

We need to have links out of the notebook onto the rest of the web where we have content. This should also be done, but needs to have its generalization tested.

## Tasks
- [x] (`🔑Required`) Add buttons for each platform with helpful icons and text.
- [x] (`🔑Required`) **Blocked on 01/07:** Guarantee that `autobot` will make the appropriate translations from the `syllabus.yml` to each page.
- [x] (`🍜 Nice to have`) Potentially re-color the buttons to match the brand identity of each platform? This might help with visual grepping, just like the icons.

Use the colors specified here: https://brandcolors.net/

**Comment by ionlights** [comment]
@brandons209 added so you can see this progress since it's something you requested we address.

**Comment by ionlights** [comment]
Current status: Buttons have been added.

What's left:
- [ ] Guarantee that `autobot` will make the appropriate translations from the `syllabus.yml` to each page.
- [ ] (`🍜 Nice to have`) Potentially re-color the buttons to match the brand identity of each platform? This might help with visual grepping, just like the icons.
